sys-process/runit/runit-2.2.0.ebuild

```diff
@@ -50,22 +48,39 @@ src_install() {
 	dobin $(<../package/commands)
 	dodir /sbin
 	mv "${ED}"/usr/bin/{runit-init,runit,utmpset} "${ED}"/sbin/ || die "dosbin"
-	if use split-usr ; then
-		dosym ../etc/runit/2 /sbin/runsvdir-start
+	if use scripts ; then
+		if use split-usr ; then
```

Comment:
This `if` block installs symlink `/sbin/runsvdir-start` which points to `/etc/runit/2`. This is harmless even with USE=-scripts - that symlink can be installed even if there is no `/etc/runit/2` or if `/etc/runit/2` was created by user manually. So, we probably won't need `if use scripts` here.

TBH I doubt this symlink has any actual use at all. The name `/sbin/runsvdir-start` comes from a recommendation at https://smarden.org/runit/useinit and chances are no Gentoo users are using it. But we can keep the symlink for compatibility reasons, just in case someone uses it.

Reply:
> This `if` block installs symlink `/sbin/runsvdir-start` which points to `/etc/runit/2`. 
I'd like to keep runsvdir-start still available when USE="-script".